static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-form-inline-footer.js

```diff
@@ -56,29 +58,29 @@ define([
         };
 
         this.id = 'RtrtEmailFormInlineFooter';
```

Comment:
That is a very good point, everyone who has visited recently will end up with notintest in their localstorage. I'll change the test name, thanks!